css/mirador.css

```diff
@@ -1538,6 +1538,11 @@ div.manifest-info a.mirador-btn.mirador-icon-toc {
   max-height: 185px;
   overflow: auto;
 }
+
+.qtip-header {
+   background: #cecede;
+}
```

Comment:
This sets a background for the header - it looks kind of empty; I wish I could find a non-hacky way of pushing the first edit and delete buttons into the header.